config/src/main/java/org/hyperledger/besu/config/JsonQbftConfigOptions.java

```diff
@@ -34,6 +36,11 @@ public JsonQbftConfigOptions(final ObjectNode bftConfigRoot) {
     return JsonUtil.getString(bftConfigRoot, VALIDATOR_CONTRACT_ADDRESS).map(String::toLowerCase);
   }
 
+  @Override
+  public OptionalLong getStartBlock() {
```

Comment:
What does empty startBlock represent? Seems a bit odd and I am wondering if instead we might be better defaulting to zero. That way startBlock has a consistent meaning.